test/single-layer/test_layers.cpp

```diff
@@ -0,0 +1,171 @@
+#include <algorithm>
+#include <vector>
+
+#include "gtest/gtest.h"
+#include "layers/EWLayer.hpp"
+#include "layers/FCLayer.hpp"
```

Comment:
Split this file into 2: EW and FC layers

Reply:
Splitted